### PR TITLE
Fix SessionApiTest for real

### PR DESCRIPTION
**Pull request by bidetofevil**
## Goal
Fix SessionApiTest by actually removing the expectation of the disk space attribute smdh

### CODE REVIEW

**Comment by bidetofevil** [comment]
* **#1418** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1418?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bidetofevil and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/SessionApiTest.kt [hunk 1]
```diff
@@ -73,7 +73,6 @@ internal class SessionApiTest {
                 "emb.session_number" to "1",
                 "emb.type" to "ux.session",
                 "emb.error_log_count" to "0",
-                "emb.disk_free_bytes" to "0",
```
[the remark this answers is not in the document]
Comment (bidetofevil) [reply embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/SessionApiTest.kt:76]: LOL, yup yup, Realized this. That's the nature of fixing flakey tests - you think you fixed it, but it was just flakey

**Review by fractalwrench — APPROVED** [review]
LGTM

**Comment by bidetofevil** [comment]
### Merge activity

* **Sep 24, 12:07 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1418) rebased this pull request as part of a merge.

**Pull request merged**